api/src/utils/error-replacer.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Extract values from Error objects for use with JSON.stringify()
+ */
+export function errorReplacer(_key: string, value: any) {
```

Comment:
Can we come up with a bit better name for this? Looks like it's trying to standardize the stringified output of an Error object? The name `errorReplacer` isn't descriptive enough 🙂

Reply:
How does `jsonStringifyErrorReplacer` sound? 👀 
It's used as `JSON.stringify()`'s second arg.

Reply:
It looks to me like it basically transforms an error object into a typed object, so maybe we call it something like `errorToSanitizedObject` or something? 🤔

Reply:
The choice of name was because it's known as `replacer function` and it is supposed to be used alongside `JSON.stringify()` only. Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify

> maybe we call it something like `errorToSanitizedObject` or something

The function isn't sanitizing as the result of error object after `JSON.stringify()` is `{}` due to it's `enumerable` flag being `false`. Hmm, or should the `errorReplacer()` be exported from the redact util since it's only used there right now?

Reply:
👍 Moved it in https://github.com/directus/directus/pull/18777/commits/275ed152df4ccfe559dd574578c096a6b153df02